Jenkinsfile

```diff
@@ -27,6 +20,11 @@ node('ubuntu-zion') {
   GitHub gitHub
 
   try {
+    stage('Init IQ Version & Sha') {
+      nexusIqVersion = getVersionFromBuildName(env.releaseBuild_NAME)
+      nexusIqSha = readBuildArtifact('insight/insight-brain/release', env.releaseBuild_NUMBER,
+        "artifacts/nexus-iq-server-${nexusIqVersion}-bundle.tar.gz.sha256")
```

Comment:
Maybe fail the build here if the version and sha aren't available rather than checking for the presence of those variables later.

Reply:
> Maybe fail the build here if the version and sha aren't available rather than checking for the presence of those variables later.

The `getVersionFromBuildName` library  fails [here](https://github.com/sonatype/jenkins-shared/blob/b4161c5d454d5f2aa2cb129cdb328d06a6f78d84/vars/getVersionFromBuildName.groovy#L24). I've raised a [PR](https://github.com/sonatype/jenkins-shared/pull/140) for reading sha (build artifact). That will also fail [here](https://github.com/sonatype/jenkins-shared/blob/069bf7fd2124b34fd3c4c0c3e38dc23b16ec1abe/vars/readBuildArtifact.groovy#L15). 
Removed checks with 7a7dcc23